src/components/ae-icon/ae-icon.md

```diff
@@ -43,6 +43,7 @@
     <ae-icon fill="secondary" name="back" />
     <ae-icon fill="neutral" name="active" />
     <ae-icon fill="alternative" name="copy" />
+    <ae-icon fill="white" name="copy" />
```

Comment:
This icon is not visible in docs because it's white.

Reply:
Fixed.